client/js/screens/Campaign/Campaign.js

```diff
@@ -97,12 +100,7 @@ const Campaign = ({navigation, user, campaigns}) => {
       <View>
         <Text style={styles.campaignCategoryTitle}>Events</Text>
         {navigation.state.params.campaign.events.length > 0 ? (
-          <FlatList
-            style={styles.eventSingle}
-            data={navigation.state.params.campaign.events}
-            renderItem={({item}) => <ListEvent event={item} />}
-            keyExtractor={(item, index) => 'index' + index.toString()}
-          />
+          <>{events}</>
```

Comment:
Instead of mapping your component, keep using FlatList and use the `ListHeaderComponent` prop to render the header of your screen. Create a function that returns the header component and call it from the   `ListHeaderComponent` prop.

https://stackoverflow.com/questions/58243680/react-native-another-virtualizedlist-backed-container

https://github.com/GeekyAnts/NativeBase/issues/2947

Reply:
Can't use flatlist inside scrollview

Reply:
That's why you have to remove the ScrollView and use the `ListHeaderComponent` prop to show the header of your screen.